src/lib/viewers/controls/color-picker/ColorPickerToggle.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import ColorPickerPalette from './ColorPickerPalette';
+import { AnnotationMode } from '../annotations/types';
+import './ColorPickerToggle.scss';
+
+export type Props = {
+    isActive?: boolean;
+    annotationMode?: AnnotationMode;
+};
+
+export default function ColorPickerToggle({ annotationMode, isActive = false, ...rest }: Props): JSX.Element | null {
```

Comment:
This naming is very confusing, since a "toggle" in other controls is just a button. Maybe `ColorPickerControl` or something to that effect?

Reply:
Yes, I think that makes sense.